src/Navigation.js

```diff
@@ -81,8 +84,6 @@ const HomeStack = StackNavigator(
   {
     initialRouteName: HOME,
     navigationOptions: {
-      tabBarIcon: tabIcon(iconHomeDefault, iconHomeActive),
-      tabBarLabel: text.tabHome,
       tabBarTestIDProps: {
```

Comment:
This needs to be moved down into the tabBarNavigator props (like the others). I think this will help the e2e tests pass.

Reply:
Done!